app/models/script.rb

```diff
@@ -0,0 +1,24 @@
+class Script < ApplicationRecord
```

Comment:
Per my comment in https://github.com/Ragtagteam/cta-aggregator-docs/pull/2#issuecomment-290888675

What do you think about maybe renaming the model to `CallScript`? I worry about cognitive collision between this and any scripting we do in the app.